Client/Client.Core/Client.Core.Static/Client.hpp

```diff
@@ -11,174 +11,283 @@
+    */ 
     void askForChannelList() const;
-    /// Ask for channel subscription list
+
+    /**
+    * @brief Ask for channel subscription list.
+    */ 
     void askForChannelSubscriptionList() const;
+
     /**
-     * @brief Ask for channel message history
-     * @param channelID Channel ID
-     */
+    * @brief Ask for channel message history
+    * @param channelID Channel ID
+    */
     void askForMessageHistory(uint64_t channelID) const;
+
     /**
-     * @brief Ask for reply message history
-     * @param channelID
-     */
+    * @brief Ask for reply message history
+    * @param channelID
+    */
     void askForReplyHistory(uint64_t channelID) const;
+
     /**
-     * @brief Sends a none message to all clients
-     */
+    * @brief Sends a none message to all clients
+    */
     void messageAll() const;
+
     /**
-     * @brief Delete user's message
-     * @param messageId message ID
-     */
+    * @brief Delete user's message
+    * @param messageId message ID
+    */
     void userMessageDelete(const std::uint64_t messageID) const;
+
     /**
-     * @brief Delete user's message
-     * @param messageId message's text
-     */
+    * @brief Delete user's message
+    * @param messageId message's text
+    */
     void userMessageDelete(const std::string& messageText) const;
+
     /**
-     * @brief Sending the ID of the subscribed channel
-     * @param message ID uint64_t variable
-     */
+    * @brief Sending the ID of the subscribed channel
+    * @param message ID uint64_t variable
+    */
     void subscriptionChannel(const std::uint64_t channelID) const;
+
     /**
-     * @brief Sending the name of the leave channel
-     * @param channel name std::string variable
-     */
-    void leaveChannel(const std::string channelName) const;
-    /**
-     * @brief Sending the name of the delete channel
-     * @param channel name std::string variable
-     */
-    void deleteChannel(const std::string channelName) const;
-    /**
-     * @brief Sending the info of the created channel
-     * @param channel name std::string variable
-     */
-    void createChannel(const std::string channelName) const;
-    /**
-     * @brief Send a message to server
-     * @param message Message
-     * @param channelID Channel ID
-     */
+    * @brief Sending the name of the leave channel
+    * @param channel name std::string variable
+    */
+    void leaveChannel(const std::string_view& channelName) const;
+
+    /**
+    * @brief Sending the name of the delete channel
+    * @param channel name std::string variable
+    */
+    void deleteChannel(const std::string_view& channelName) const;
+
+    /**
+    * @brief Sending the info of the created channel
+    * @param channel name std::string variable
+    */
+    void createChannel(const std::string_view& channelName) const;
+
+    /**
+    * @brief Send a message to server
+    * @param message Message
+    * @param channelID Channel ID
+    */
```

Comment:
Should I comment every line?
Documentation is very poor, in future we could get a lot of problems with client implementation

Reply:
This is old documentation that was written before my time...

Reply:
Yeah, but you right now not deal with any difficult tusk, so why not to try figure out what this mean...